Sources/MapboxMaps/Style/Types/Expression.swift

```diff
@@ -17,20 +17,34 @@ public struct Expression: Codable, CustomStringConvertible, Equatable {
 
     /// The operator of this expression
     public var `operator`: Operator {
-        guard let first = elements.first, case Element.operator(let op) = first else {
-            fatalError("First element of the expression is not an operator.")
+        var firstElement = elements.first
+        /// If the expression starts with an argument instead of an operator, return the first operator of a contained expression if available.
+        if case let .argument(firstArgument) = firstElement,
+           case let .expression(firstExpression) = firstArgument {
+            firstElement = firstExpression.elements.first
```

Comment:
```suggestion
            return firstExpression.operator
```
Use recursion to support expressions of any depth?

Reply:
Using this approach I think that makes sense -- will commit this suggestion.